scripts/openssl_compile.bat

```diff
@@ -38,7 +40,7 @@ ECHO Compiling openssl...
 
 pushd %1
 
-perl Configure VC-WIN64A --release --prefix=%BUILDDIR% --openssldir=%BUILDDIR%\SSL no-tests
+perl Configure VC-WIN64A --release --prefix=%BUILDDIR% --openssldir=%BUILDDIR% no-tests -DOPENSSL_NO_ENGINE
```

Comment:
Can we use some default openssldir? such as C:\Program Files\OpenSSL-Win64 ? or something similar to make it even harder setting a custom config file?